src/value.cc

```diff
@@ -652,6 +671,8 @@ bool V8ToGIArgument(GIBaseInfo *gi_info, GIArgument *arg, Local<Value> value) {
     }
     case GI_INFO_TYPE_INTERFACE:
         arg->v_pointer = GObjectFromWrapper(value);
+        if (transfer == GI_TRANSFER_EVERYTHING)
+            g_object_ref(arg->v_pointer);
```

Comment:
And this one isn't going to leak anything? Is this case the one tested by the test down here?